### PR TITLE
Sample Files Reorganization

### PR DESCRIPTION
**Pull request by jorg3lopez**
# Sample Files Reorganization

- Added sample files for MN: This PR adds two sample files to the example folder. The translation file was created by feeding the MN ORU_RO1 Lab Message to the RS API. Then we fed this FHIR file to the RS API, to get the ORU_R01 Lab Result HL7 file that will be routed to the final destination (hospital-receiver/client). These files represent the following process: RS initial ingestion -> FHIR Translation routed to TI -> FHIR Translation file sent back to RS -> HL7 file routed to final destination
- Added sample files for TN and TX
- Also, the translation file has a size of 533kb. We have a pre-commit hook that caps the max size of files to 500kb. In order to add this file to the remote repo, I had to change the max size from it's default of 500kb to 700kb.
- Added readme file with the file list
- Changed top level folders to be states and organizations
- Created a new naming convention and renamed all sample files

## Renamed files

- `fhir/MN NBS FHIR Order Message.json` => `MN/001_MN_Order_NBS.fhir`
- `fhir/lab_order.json` => `Other/002_Order.fhir`
- `fhir/newborn_patient.json` => `Other/003_Patient_NBS.fhir`
- `fhir/oml_message.json` => `Other/004_OML_O21.fhir`
- `hl7/AL/Baptist ORM 1.txt` => `Oracle/001_Oracle_ORM_O01.hl7`
- `hl7/AL/Baptist ORM 2.txt` => `Oracle/002_Oracle_ORM_O01.hl7`
- `hl7/AL/Baptist ORU.hl7` => `Oracle/004_Oracle_ORU_R01.hl7`
- `hl7/CA/CA NBS OML_O21 Lab Order Message.txt` => `CA/001_CA_OML_O21.hl7`
- `hl7/CA/CA NBS ORU_R01 Lab Result Message.txt` => `CA/002_CA_ORU_R01.hl7`
- `hl7/LA/Ochsner ORM.txt` => `Epic/001_Epic_ORM_O01.hl7`
- `hl7/LA/Ochsner ORU.txt` => `Epic/002_Epic_ORU_R01.hl7`
- `hl7/MN/ADT_A01.txt` => `MN/005_MN_ADT_A01.hl7`
- `hl7/MN/MN NBS OML_O21 Lab Order Message.txt` => `MN/002_MN_OML_O21_NBS.hl7`
- `hl7/MN/MN NBS ORM_O01 Lab Order Message.txt` => `MN/003_MN_ORM_O01_NBS.hl7`
- `hl7/MN/MN NBS ORM_O01 Lab Order Message Extended.txt` => `MN/003_MN_ORM_O01_NBS_extended.hl7`
- `hl7/MN/MN NBS ORU_R01 Lab Result Message.txt` => `MN/004_MN_ORU_R01_NBS.hl7`
- `hl7/MN/ORM simplified.txt` => `MN/003_MN_ORM_O01_NBS_simplified.hl7`
- `hl7/OML_021.hl7` => `NewSTEPs/001_NewSTEPs_OML_021.hl7`
- `hl7/ORU_R01.hl7` => `NewSTEPs/002_NewSTEPs_ORU_R01.hl7`
- `hl7/ADT_A01.txt` => `Other/001_ADT_A01.hl7`

## Issue
#640

### CODE REVIEW

**Comment by basiliskus** [comment]
When I did the translation to fhir of the same MN hl7 I get the same results with the exception of one section that is missing from the file you added to this PR. I wonder why. I have an `RelatedPerson` that comes from the `NK1` segment

**Review by saquino0827 — APPROVED** [review]

**Comment by jorg3lopez** [comment]
> When I did the translation to fhir of the same MN hl7 I get the same results with the exception of one section that is missing from the file you added to this PR. I wonder why. I have an `RelatedPerson` that comes from the `NK1` segment

Good observation. RS just had an update in their main repo that adds the NK1 segment to the file.

File: examples/hl7/MN/0001 - MN NBS ORU_R01 Lab Result Message for Initial Ingestion.txt
Comment (JohnNKing) [inline examples/hl7/MN/0001 - MN NBS ORU_R01 Lab Result Message for Initial Ingestion.txt:1]: Perhaps rename to *.hl7? (for consistency with the outgoing HL7 sample)

Comment (basiliskus) [reply examples/hl7/MN/0001 - MN NBS ORU_R01 Lab Result Message for Initial Ingestion.txt:1]: Good point. I can do that

**Review by saquino0827 — COMMENTED** [review]
Removed Approved status and pending the new file changes

**Review by saquino0827 — COMMENTED** [review]
A small nitpick item but I noticed the file names below have the <HL7_Type> and <SUB_TYPE> in a different order of the read me. Either way works for me as long as they match.
Example below:`002_MN_NBS_OML_O21.hl7`
Format in read me:`<XXX>_<SS>_<HL7_Type>_<SUB_TYPE>_<DESCRIPTION>.[fhir|hl7]`

**Comment by basiliskus** [comment]
> A small nitpick item but I noticed the file names below have the <HL7_Type> and <SUB_TYPE> in a different order of the read me. Either way works for me as long as they match. Example below:`002_MN_NBS_OML_O21.hl7` Format in read me:`<XXX>_<SS>_<HL7_Type>_<SUB_TYPE>_<DESCRIPTION>.[fhir|hl7]`

Nice catch. Will fix that

**Review by saquino0827 — APPROVED** [review]

**Comment by jcrichlake** [comment]
Do we want to have vendor specific names in the repo? Or should those change into the state and entity that they are a part of? For example LA_PHL_VENDOR or something ?

**Comment by basiliskus** [comment]
> Do we want to have vendor specific names in the repo? Or should those change into the state and entity that they are a part of? For example LA_PHL_VENDOR or something ?

That's a good question. My understanding is that the vendor and the state may not match one to one and we may want to keep them separate, but I'm not sure. I'll bring that question up in scrum today

**Comment by basiliskus** [comment]
> > Do we want to have vendor specific names in the repo? Or should those change into the state and entity that they are a part of? For example LA_PHL_VENDOR or something ?
> 
> That's a good question. My understanding is that the vendor and the state may not match one to one and we may want to keep them separate, but I'm not sure. I'll bring that question up in scrum today

@jcrichlake  confirmed in scrum that we want the states and vendors separate for now. We can change it in the future if needed

**Pull request merged**